app/javascript/components/bathroom.jsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Query } from 'react-apollo';
 import gql from 'graphql-tag';
 import styled from 'styled-components';
 import { WhiteSubTitle, WhiteTitleLarge } from '@components/typography';
 import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+// import withFragment from './hocs/with-fragment';
+import withFragmentModify from './hocs/with-fragment-modify';
```

Comment:
Lets change the definition of  the existing `withFragment` module rather than adding a new one with a slightly different name.

Reply:
Yes, this is just a draft PR and does not yet contain these fixes. Creating a new HOC was for testing purposes for the bathroom component only. I have a working version of `withFragment` and am implementing it with the remaining components.